RN_Components/src/components/play-pause/PlayPause.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { View, TouchableOpacity } from 'react-native';
+import Icon from 'react-native-vector-icons/MaterialCommunityIcons'
+
+import { styles } from './playPause-styles';
+
+interface PlayPauseProps {
+  setPaused: React.Dispatch<React.SetStateAction<boolean>>;
+  paused: boolean;
+}
+
+const PlayPause = ({ setPaused, paused }) => {
```

Comment:
don't destructure directly, destructure it from props and use the interface as well.